### PR TITLE
Context config

### PR DESCRIPTION
**Pull request by waTeim**
A modification to allow /context to export environment variables generically.  Guard against leaking information by requiring a special whitelist env variable **EXPORTABLE_ENV** which will contain a colon-separated list of environment variable names that are to be exported.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,5 +14,5 @@ python3-openid==3.1.0
 requests==2.20.1
 requests-oauthlib==1.3.0 
 selenium==3.141.0 
-tycho-api==1.0.22
+tycho-api==1.2.dev2
```
Comment (ismagilani) [inline requirements.txt]: Are there corresponding tycho_api changes? Why the change in version?

**Review by ismagilani — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (waTeim) [reply requirements.txt]: Good question, it might have crept in from other work, or perhaps that's what it was like when I branched and it reverted.  In either case, the changes are not dependant on tycho.
